app/controllers/publish/courses/study_sites_controller.rb

```diff
@@ -38,10 +38,19 @@ def update
         end
       end
 
+      def back
+        authorize(@provider, :edit?)
+        if @provider.study_sites.count > 1
```

Comment:
could be `@provider.study_sites.many?`

if the length doesn't upset rubocop potentially even...
`redirect_to @provider.study_sites.many? ? new_publish_provider_recruitment_cycle_courses_study_sites_path(path_params) : @back_link_path`

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3671/commits/4731aa660968b9bfce61eb872d1d37f401f1c5d0